src/common/SignalVectorModel.hpp

```diff
@@ -182,7 +182,7 @@ class SignalVectorModel : public QAbstractTableModel,
 
             QVector<int> roles = QVector<int>();
             roles.append(role);
-            emit dataChanged(index, index, roles);
+            dataChanged(index, index, roles);
```

Comment:
It would be great if these could be prefixed with `this->` to indicate that they're class-methods.